public/src/components/common/nav_btn_group.jsx

```diff
@@ -48,11 +49,19 @@ export default class DashboardButtonGroup extends Component {
           </ButtonGroup>
         ) : (
           <ButtonGroup>
-            <Button type="button" onClick={this.handleAvailCampaignsClick}>
-              Available Campaigns
+            <Button
+              type="button"
+              onClick={this.handleJoinedCampaignsClick}
+              className={pathname && pathname === '/volunteers/campaigns' ? 'active' : ''}
```

Comment:
This is a nice touch - just maybe in the future we can find a way to make this kind of logic more reusable.

Reply:
sounds good, I agree. I noticed React Router has a cool `<NavLink />` component we can also use and add the active class name there: https://reacttraining.com/react-router/web/api/NavLink